templates/vm.xml.j2

```diff
@@ -44,6 +44,7 @@
       <target dev={{ volume.target }} />
       {% endif %}
     </disk>
+    <rng model="virtio"><backend model="random">/dev/urandom</backend></rng>
```

Comment:
This should not be inside the volume for loop.

Reply:
true! thank you for reviewing this.